docs/usage.md

```diff
@@ -201,7 +201,7 @@ const sequelize = new Sequelize('sqlite:relativePath/dbname.db')
 
 ### PostgreSQL
 
-The library for PostgreSQL is`pg@~3.6.0` You'll just need to define the dialect:
+The library for PostgreSQL is`pg@~6.1.0` You'll just need to define the dialect:
```

Comment:
I don't see anything wrong with documenting that both 6 and 7 are supported. Otherwise someone will have to update this documentation once 7 is out